src/zenml/models/step_run_models.py

```diff
@@ -251,7 +250,10 @@ class StepRunRequestModel(StepRunBaseModel, WorkspaceScopedRequestModel):
         title="Logs associated with this step run.",
         default=None,
     )
-    deployment: "PipelineDeploymentResponseModel"
+    deployment: UUID = Field(
```

Comment:
```suggestion
    deployment_id: UUID = Field(
```

Reply:
Oh, yes, you're right. I remember all of them were `X_id` at some point so I thought that was our convention. But apparently this was changed sometime.